### ISSUE TITLE
Debug *.js.dna files with sourcemaps

### ISSUE THREAD

**Issue by xekoukou** [sustrik/ribosome#39]
http://www.html5rocks.com/en/tutorials/developertools/sourcemaps/

Now, the only way seems to  be to generate the rna file and debug that file.

**Comment by sustrik** [comment]
Yes. But I don't think there's an easy solution for that. Every compiler (and ribosome is a compiler) can either require generating the output and using the underlying language's debugging tools on that or build its own debugging tools.

**Comment by xekoukou** [comment]
I am putting this as a future goal. I haven't looked at it very closely.

The way to debug node-javascript is to use the browser debugger (like chrome debugger). Chrome supports sourcemaps. So it seems that we just need to generate the sourcemaps. 

I haven't looked how to do it but javascript has quite a big community that makes me optimistic.

For reference, I need to look at these:
https://www.npmjs.com/package/source-map
https://hacks.mozilla.org/2013/05/compiling-to-javascript-and-debugging-with-source-maps/
